stade/settings.py

```diff
@@ -26,7 +26,8 @@ def before_binding(configuration: ComposedConfiguration) -> None:
         auth_app_index = configuration.INSTALLED_APPS.index(
             'composed_configuration.authentication.apps.AuthenticationConfig'
         )
-        configuration.INSTALLED_APPS.insert(0, 'jazzmin')
+        admin_index = configuration.INSTALLED_APPS.index('django.contrib.admin')
```

Comment:
Reorder these lines. The first insert could change the effective position of the second.